igcollect/ping.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
```

Comment:
i would expect some comments up here that describes the purpose of the script and maybe some usage examples